src/pages/index.js

```diff
@@ -90,7 +182,7 @@ addCardButton.addEventListener('click', () => {
   addCardPopup.open();
   addCardValidator.resetValidation();
 });
-previewModal.addEventListener('click', () => imagePreviewPopup.open());
-
+//previewModal.addEventListener('click', () => imagePreviewPopup.open());
+profileAvatarEdit.addEventListener('click', () => avatarPopup.open());
```

Comment:
Please, call `resetValidation` for the avatar form as well to disable the button